app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,105 @@
+class TweetsController < ApplicationController
+    
+  get '/tweets/new' do
+    if logged_in
+      erb :'/tweets/new'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if logged_in
+      @user = current_user
+      
+      if params[:content].length <= 140
+        
+        @tweet = Tweet.new(content: params[:content], user_id: @user.id)
```

Comment:
`current_user.tweets.build(params)` is the best way to do this. It creates an associative relationship for us already.